schema/base/bathymetry.yaml

```diff
@@ -0,0 +1,23 @@
+---
+"$schema": https://json-schema.org/draft/2020-12/schema
+title: bathymetry
+description: topographic representation of the ocean floor 
+type: object
+properties:
+  id:
+    "$ref": ../defs.yaml#/$defs/propertyDefinitions/id
+  geometry:
+    unevaluatedProperties: false
+    oneOf:
+      - "$ref": https://geojson.org/schema/Polygon.json
+      - "$ref": https://geojson.org/schema/MultiPolygon.json
+  properties:
+    unevaluatedProperties: false
+    allOf:
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/overtureFeaturePropertiesContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/levelContainer
+      - "$ref": ../defs.yaml#/$defs/propertyContainers/cartographyContainer
+    required:
+      - elevation
+    properties:
+      elevation: { "$ref": ./defs.yaml#/$defs/propertyDefinitions/elevation }
```

Comment:
Comments made in schema meeting 8/21 - shouldn't this be `depth`?

Reply:
updated this!